app/src/main/res/layout/license_text_viewer_fragment.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+
+  <data>
+
+    <variable
+      name="viewModel"
+      type="org.oppia.android.app.help.thirdparty.LicenseTextViewModel" />
+  </data>
+
+  <ScrollView
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+      android:id="@+id/copyright_license_text_view"
```

Comment:
Please add color, font and font-size attributes also.

Reply:
@rt4914 What should be the ideal values for these attributes?

Reply:
Added these attributes, please let me know if there should be some different values for some attributes.

Reply:
Use style either `Body` or `Subtitle2` from `styles.xml` file.